src/app/weeks/weeks.component.ts

```diff
@@ -0,0 +1,89 @@
+import { Component, OnInit } from '@angular/core';
+import { Apollo, ApolloQueryObservable } from 'apollo-angular';
+import { sortBy } from 'lodash';
+import gql from 'graphql-tag';
+
+import { WeekService } from '../week.service';
+
+@Component({
+  selector: 'sc-weeks',
+  templateUrl: './weeks.component.html',
+  styleUrls: ['./weeks.component.less']
+})
+export class WeeksComponent implements OnInit {
+
+  // we don't use thisweek but if we don't request it then new weeks will
+  // never be created in the db.
+  private weeksGql = gql`
+    query {
+      thisweek {
```

Comment:
You don't use thisweek ?

Also note that it won't match up with the frontend's 'current week'